### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by AKST** [ITatJCU/hunter-server#28]
Like Auth tokens that you pass in the Authorisation head or whatever, or can anyone update the api from any computer?

If it's the latter, maybe we should consider to implement that later down the track. Otherwise I could probably just inspect the application get my user id and then check the JSON for each player to get the scans and make http requests to the server with my id + the scan id?

**Comment by axrs** [comment]
Correct, no authentication is currently implemented. We don't want users to
create accounts, so it's never been that vital.

––Sent by phone. Xander

**Comment by AKST** [comment]
In the mean time I'll just keep a copy of the auth token on the client side.

**Comment by AKST** [comment]
We could generate a token in the initialisation of their cookie/whatever and they would lose it on clearing that data, that way they don't have to create an account and you get a basic level of security. But I guess we'd need to add `https` as well to make it worth while otherwise people could sniff the traffic and get the auth token anyways.

**Comment by AKST** [comment]
Either way people aren't like to go out their way to game the system, but it would be an interesting extension to the project
